### ISSUE TITLE
GLib-GObject-CRITICAL **: 08:35:12.915: g_object_ref: assertion 'old_val > 0' failed

### ISSUE THREAD

**Issue by loganthomas** [Homebrew/homebrew-core#38138]
**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [X] are reporting a bug others will be able to reproduce and not asking a question or requesting software. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh. To get software added or changed in Homebrew please file a [Pull Request](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)
- [X] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, Homebrew/homebrew-core formula (not cask) on macOS? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a Linux problem please file this issue at https://github.com/Homebrew/linuxbrew-core/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [X] ran `brew update` and can still reproduce the problem?
- [X] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [X] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [X] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:
- What you were trying to do (and why)
- What happened (include command output)
- What you expected to happen
- Step-by-step reproduction instructions (by running `brew install` commands)

## General
I have a python script that leverages WeasyPrint and Jinja2. When I try to run my script that writes multiple HTML files to PDF files, I am getting the below error message:

```python
(process:8363): GLib-GObject-CRITICAL **: 08:35:12.915: g_object_ref: assertion 'old_val > 0' failed
Segmentation fault: 11
```

## Specs
- Mac OS X 10.14.3
- python 3.7
- glib 2.60.0
- brew config
```
HOMEBREW_VERSION: 2.0.5
ORIGIN: https://github.com/Homebrew/brew
HEAD: 417a16fbafa97a5740d355ed33bfbfd2823452e7
Last commit: 4 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: c30aa75391fbc773cb987fb453894594d5149a1d
Core tap last commit: 4 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_LOGS: /Users/logan/Library/Logs/Homebrew
CPU: dodeca-core 64-bit kabylake
Homebrew Ruby: 2.3.7 => /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/bin/ruby
Clang: 10.0 build 1000
Git: 2.17.2 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.54.0 => /usr/bin/curl
macOS: 10.14.3-x86_64
CLT: 10.1.0.0.1.1539992718
Xcode: N/A
```

- brew doctor
```
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and what additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew provided
script of the same name. We found the following "config" scripts:
  /anaconda3/bin/icu-config
  /anaconda3/bin/freetype-config
  /anaconda3/bin/xslt-config
  /anaconda3/bin/libpng16-config
  /anaconda3/bin/python3.7-config
  /anaconda3/bin/libpng-config
  /anaconda3/bin/xml2-config
  /anaconda3/bin/python3.7m-config
  /anaconda3/bin/python3-config
  /anaconda3/bin/curl-config
  /anaconda3/bin/ncursesw6-config
  /anaconda3/bin/pcre-config
```

## Example
```python
from weasyprint import HTML
```
```html
html = """
<!DOCTYPE html>
<html lang="en">

<head>
  <title>Title of doc</title>
  <meta charset="UTF-8">
</head>

<body>
	<div style="text-align:center">
	    <img src="pic.jpg" alt="pic" style="height:500px;width:500px;">
	</div>

	<div style="text-align:center">
	    <h1>PDF Title Page</h1>
	    <h2><i>Subtitle</i></h2>
	</div>
        <p style="page-break-before: always"></p>
        <h1>Page #2</h1>

</body>
</html>
""" 
```
```python
for i in range(5):
    HTML(string=html).write_pdf(target=f'/tmp/test-{i}.pdf')
```

Only test-0.pdf gets written and I get kicked out of my python session.

```python
# Output
(process:8363): GLib-GObject-CRITICAL **: 08:35:12.915: g_object_ref: assertion 'old_val > 0' failed
Segmentation fault: 11
```

**Comment by SMillerDev** [comment]
Does the same thing happen with the python version that homebrew ships?

**Comment by loganthomas** [comment]
THANK YOU!!! I was assuming my python 3.7 was that same as the python version that homebrew shipped with. After some investigation, I saw I had python 3.7.0 and homebrew shipped with python 3.7.2.

I created a virtual env with python 3.7.2 and all worked fine!!!
